recipe/meta.yaml

```diff
@@ -53,12 +56,12 @@ test:
     - python -m daal4py examples/sycl/sklearn_sycl.py
 
 about:
-  home: https://intelpython.github.io/daal4py/
+  home: https://intel.github.io/scikit-learn-intelex/
```

Comment:
Don't think so - this is daal4py package and we should point to daal4py documentation

Reply:
Thanks!